multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/heancms/HeanCms.kt

```diff
@@ -117,6 +117,16 @@ abstract class HeanCms(
 
     override fun latestUpdatesParse(response: Response): MangasPage = popularMangaParse(response)
 
+    override fun fetchSearchManga(page: Int, query: String, filters: FilterList): Observable<MangasPage> {
+        if (!query.startsWith(SEARCH_PREFIX)) {
+            return super.fetchSearchManga(page, query, filters)
+        }
+
+        return fetchMangaDetails(SManga.create().apply { url = "/series/${query.substringAfter(SEARCH_PREFIX)}" }).map {
+            MangasPage(listOf(it), false)
+        }
```

Comment:
Use this to increase the legibility.

```suggestion
        val slug = query.substringAfter(SEARCH_PREFIX)
        val manga = SManga.create().apply { url = "/series/$slug" }
        
        return fetchMangaDetails(manga).map { MangasPage(listOf(it), false) }
```